packages/client/src/components/instance-ticker.vue

```diff
@@ -14,14 +15,16 @@ const props = defineProps<{
 		faviconUrl?: string
 		name: string
 		themeColor?: string
+		softwareName?: string
```

Comment:
I noticed there are no semicolons after the types so I did not add one here either. Should they be added?